spec/outputs/jms_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "logstash/devutils/rspec/spec_helper"
+require 'logstash/outputs/jms'
+require 'jms'
+require 'json'
+
+def getYamlPath()
+  return File.join(File.dirname(__FILE__),"jms.yml")
+end
+
+describe "outputs/jms" do
+  let (:jms_config) {{'yaml_file' => getYamlPath(), 'yaml_section' => 'hornetq', 'destination' => 'ExampleQueue'}}
+  let (:event) { LogStash::Event.new({'message' => 'hello',
+                                      '@timestamp' => LogStash::Timestamp.now}) }
+
+  context 'when initializing' do
+    it "should register" do
+      output = LogStash::Plugin.lookup("output", "jms").new(jms_config)
+      expect {output.register}.to_not raise_error
+    end
+
+    it 'should populate jms config with default values' do
+      jms = LogStash::Outputs::Jms.new(jms_config)
+      insist {jms.pub_sub} == false
```

Comment:
You can use expect with something like expect(...).to eq(false) and use the same notations always :-)